contracts/account/extensions/AccountERC7579.sol

```diff
@@ -164,6 +163,30 @@ abstract contract AccountERC7579 is
         return _execute(Mode.wrap(mode), executionCalldata);
     }
 
+    /**
+     * @dev Implement ERC-1271 through IERC7579Validator modules. If module based validation fails, fallback to
+     * "native" validation by the abstract signer.
+     *
+     * NOTE: when combined with {ERC7739} (for example through {Account}), resolution ordering may have an impact
+     * ({ERC7739} does not call super). Manual resolution might be necessary.
+     */
+    function isValidSignature(bytes32 hash, bytes calldata signature) public view virtual override returns (bytes4) {
+        // check signature length is enough for extraction
+        if (signature.length >= 20) {
+            (address module, bytes calldata innerSignature) = _extractSignatureValidator(signature);
+            // if module is not installed, skip
+            if (isModuleInstalled(MODULE_TYPE_VALIDATOR, module, Calldata.emptyBytes())) {
+                // try validation, skip any revert
+                try IERC7579Validator(module).isValidSignatureWithSender(address(this), hash, innerSignature) returns (
+                    bytes4 magic
+                ) {
+                    if (magic == IERC1271.isValidSignature.selector) return magic;
+                } catch {}
```

Comment:
To confirm, ERC-7579 is not explicit whether the function should revert. So we're basically protecting against an unexpected revert. Is this correct?

Also side note but this is the kind of code that may be better to do in assembly. For another PR